docs/developer-guide/CLI Methods/generating-the-bundled-config-file.md

```diff
@@ -0,0 +1,87 @@
+# Generating the Bundled Configuration File
```

Comment:
Can we rename the directory to `docs/developer-guide/cli-methods/bundling-basemaps-config-file.md`